### ISSUE TITLE
Not working !!!

### ISSUE THREAD

**Issue by Suleman-Elahi** [zebulgar/supertracker#1]
The pixel doesn't seem to be loading. It's only showing the sender details.

**Comment by Suleman-Elahi** [comment]
Here's no read me as well !!! Pls mention the steps to set up this on my own server.

**Comment by FearNaBoinne** [comment]
Agreed, where is that readme in the tutorial section as mentioned on the main page?

**Comment by Suleman-Elahi** [comment]
Nowhere I am afraid.
